ray-operator/apis/config/v1alpha1/configuration_types.go

```diff
@@ -0,0 +1,52 @@
+package v1alpha1
```

Comment:
Because we are introducing a new API group (config.ray.io), which is different from ray.io where RayCluster, RayJob and RayService lives. Config API is new and mostly adopts existing flags for now. I imagine we'll want to change and iterate on this API a little before it goes GA.

Reply:
Originally, we placed the CRD only in `apis/config/{v1|v1alpha1}`. Is it a common pattern to add APIs that are not CRDs to the directory (e.g., `config.ray.io`)?

Reply:
yes, you can have any Kubernetes-like API in `apis/`. The generator does not care or know if it's a CRD or not, it just looks at the types. 

See Kueue as another example that has a config API: https://github.com/kubernetes-sigs/kueue/tree/main/apis/config/v1beta1